packages/interaction-kit/package.json

```diff
@@ -37,6 +37,7 @@
 		"discord-snowflake": "2.0.0",
 		"discord-verify": "1.0.1",
 		"dotenv": "^16.0.1",
+		"esbuild": "0.15.13",
```

Comment:
Maybe should be peerDependencies instead so that it can be installed as a devDependency